### PR TITLE
Kdump 3 types storage

### PR DESCRIPTION
**Pull request by littlestar565**
Add(Kdump): [kdump]Trigger kernel core dump through NFS under network traffic

### CODE REVIEW

File: testscripts/kdump_3_types_storage.ps1 [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+
+# Change the dir for changing crashkernel for EFI or BIOS on RHEL7/8
+$dir = "/boot/grub2/grub.cfg"
+$words = $vmName.split('-')
+if ($words[-2] -eq "EFI")
+{
+    $dir = "/boot/efi/EFI/redhat/grub.cfg"
+}
+Write-host -F Red "DEBUG: $vmName $dir"
+
```
Comment (yangboroy) [inline testscripts/kdump_3_types_storage.ps1]: Line 137 - 143, for grub file, please consider /etc/default/grub.

File: testscripts/kdump_3_types_storage.ps1 [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+    return $Aborted
+}
+# Refresh status
+$vmObjB = Get-VMHost -Name $hvServer | Get-VM -Name $vmNameB
+$IPB = GetIPv4ViaPowerCLI $vmNameB $hvServer
+
+
+# Change crashkernel
+Write-Host -F Green "INFO: Change the crashkernel = 512M of $ipv4"
+$result = bin\plink.exe -i ssh\${sshKey} root@$ipv4 "sed -i 's?crashkernel=auto?crashkernel=512M?' /etc/default/grub && grub2-mkconfig -o $dir && echo `$? "
```
Comment (yangboroy) [inline testscripts/kdump_3_types_storage.ps1]: If core of case isn't test crashkernel value, please use "auto", i think it is very stable than 512M.

Comment (littlestar565) [reply testscripts/kdump_3_types_storage.ps1]: If keep it as "auto", when the memory of VMs<=8G,  kdump will not start. So we need to config it.

Comment (yangboroy) [reply testscripts/kdump_3_types_storage.ps1]: You mean VM's memory size < 8G, crashkernel=auto won't work, right?

**Pull request merged**